docs/sources/operators-guide/visualizing-metrics/requirements.md

```diff
@@ -19,6 +19,8 @@ The following table shows the required label names and whether they can be custo
 | `pod`       | Yes          | The unique identifier of a Mimir replica (eg. Pod ID when running on Kubernetes). The label name can be configured with the `per_instance_label` field in the mixin config.                                                                                                                                                                                                                                                                                                                      |
 | `instance`  | Yes          | The unique identifier of the node or machine where the Mimir replica is running (eg. the node when running on Kubernetes). The label name can be configured with the `per_node_label` field in the mixin config.                                                                                                                                                                                                                                                                                 |
 
+For dashboards and rules to function properly, your Prometheus or Grafana Agent must be configured to scrape metrics from Grafana Mimir at a 10s interval.
```

Comment:
Problem is that many [alerts use 1m rates](https://github.com/grafana/mimir/blob/main/operations/mimir-mixin/alerts/alerts.libsonnet), and we should be using [4x of the scrape interval](https://www.robustperception.io/what-range-should-i-use-with-rate/). Since we're using 1m, scrape interval should be max `15s`.

I chose `10s` for the demo.

In dashboards we use `$__rate_interval` variable provided by Grafana, which is based on data-source configuration.

```suggestion
For rules and alerts to function properly, your Prometheus or Grafana Agent must be configured to scrape metrics from Grafana Mimir at a `15s` interval or shorter.
```